apps/api/src/server/utils/application-factory-for-tests.js

```diff
@@ -18,60 +93,32 @@ const generateApplicationReference = () => {
  * 	regions?: object[],
  *  subSectorId?: number | null,
  * 	subSector?: import('@pins/api').Schema.SubSector,
- * 	zoomLevelId?: number | null
+ * 	zoomLevelId?: number | null,
+ *  dates?: object,
+ *  inclusions?: object
  * }} arg
+=======
+ * @param {{id: number, title: string, description: string, dates: {createdAt: Date, modifiedAt: Date, publishedAt: Date}, inclusions: {serviceCustomer?: boolean, ApplicationDetails?: boolean, CaseStatus?: boolean}}} arg
+>>>>>>> 3ac242da (test(api/applications): allows case to have no info (BOAS-164))
  * @returns {import('@pins/api').Schema.Case}
  */
 export const applicationFactoryForTests = ({
```

Comment:
Now that we have changed this factory, are all the other tests that use this pass?